cartographer/mapping/imu_tracker_test.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer/mapping/imu_tracker.h"
+
+#include "cartographer/common/make_unique.h"
+#include "gtest/gtest.h"
+
+namespace cartographer {
+namespace mapping {
+namespace {
+
+const double kDuration = 3.f;
+const double kPrecision = 1e-8;
+const int kSteps = 10;
+
+class ImuTrackerTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    imu_tracker_ = common::make_unique<ImuTracker>(0.1 * kDuration, time_);
```

Comment:
Please document meaning of this constant, either with a comment on  a descriptive constant name.